### ISSUE TITLE
E13 - Page title does not identify contents (i.e. Events related to [keyword])

### ISSUE THREAD

**Issue by jpinsent** [ccswbs/hjckrrh#197]
Similar to resolved issue #146. The listing page for events filtered by keyword is titled "Upcoming Events". 

https://aoda.web.uoguelph.ca/aodademo/events/70 

While a heading does appear above the content stating which keyword is being searched, an ambiguous page title fails WCAG 2.0 criterion 2.4.2. This issue must be resolved for this feature to be considered accessible.

**Comment by mmafe** [comment]
![filtered-events-by-keyword](https://cloud.githubusercontent.com/assets/1927902/10024060/5899657e-6123-11e5-9444-4d9376ec24d1.png)
